src/Behat/Behat/Output/Node/Printer/ListPrinter.php

```diff
@@ -154,6 +149,8 @@ public function printFailedHooksList(OutputPrinter $printer, $intro, array $fail
      * @param string        $style
      * @param null|string   $stdOut
      * @param null|string   $error
+     *
+     * @deprecated Remove in 4.0
```

Comment:
I can not remove the method, because it is a dependency of the other deprecated logic. It's more of a reminder for myself to drop the guy in 4.0.